packages/webapp/src/shared/components/layout/sidebar/sidebar.component.tsx

```diff
@@ -107,8 +110,8 @@ export const Sidebar = (props: HTMLAttributes<HTMLDivElement>) => {
 
               <RoleAccess>
                 <Link
-                  className={menuItemClassName}
-                  to={generateLocalePath(RoutesConfig.subscriptions.index)}
+                  className={menuItemClassName({ isActive: pathname.includes('subscriptions') })}
```

Comment:
This looks like a hack:) I think this should work without this change.
The solution should be to change the `current subscrition` subpath to index in `app.component.tsx`